src/aquarius/Aquarius.ts

```diff
@@ -14,18 +13,22 @@ export class Aquarius {
 
   /** Resolves a DID
    * @param {string} did
-   * @param {string} fetchMethod fetch client instance
+   * @param {AbortSignal} signal abort signal
    * @return {Promise<DDO>} DDO
```

Comment:
isn't this returning `Asset` as it's coming from Aquarius?

Reply:
true, but not the issue